README.md

```diff
@@ -1,2 +1,154 @@
-# holonix
+# Holonix
+
 Holochain app development environment based on Nix.
+
+## Quickstart - Dev shell from default template
+
+hApp developers can use a Nix command to generate a flake that includes Holochain and other required binaries in a dev shell.
+
+```shell
+nix flake init -t github:holochain/holonix
```

Comment:
Maybe the `new` command makes more sense as the `init` command adds the Nix files to the current directory and so it might be confusing if a user runs it in their home directory or something.
```suggestion
nix flake new -t github:holochain/holonix my-happ
```
Or add a comment about how it (the `init` command) should be run inside the project dir.

Reply:
> should be run inside the project dir

Yes please!